src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -80,4 +82,26 @@ public static void writeToFile(Path file, String content) throws IOException {
         Files.write(file, content.getBytes(CHARSET));
     }
 
+    /**
+     * Assumes source file exists
+     */
+    public static void copyFile(Path source, Path destination) throws IOException {
+        Files.copy(source, destination);
+    }
+
+    /**
+     * Extracts extension of file
+     */
+    public static String extractExtension(Path filePath) {
+        String fileName = filePath.toString();
+        return extractExtension(fileName);
+    }
+
+    /**
+     * Extracts extension of file
+     */
+    public static String extractExtension(String fileName) {
```

Comment:
This may not work for stuff like `tar.gz`

Reply:
This should work for `asdf.tar.gz`. `.gz` will be extracted from the function and validated.